index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/d86848cfe0.js"></script>
+    <title>Sign Up</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <header>
+        <div class="header-section">
+            <p class="normal_desc">Already have an intuit account?</p>
+            <a href="#" class="normal_link">I forgot my user iD or password</a>
+        </div>
+        <a class="button" href="#"><i class="fa fa-lock" aria-hidden="true"></i> Sign in</a>
+    </header>
+    <main>
+      <div class="intuit_logo">
+          <img src="images/intuit_logo.png" alt="intuit logo" width="130">
```

Comment:
Do not include inline styling
Make sure that basic CSS bad practices are not used ([article for support](https://speckyboy.com/good-bad-css-practices) ) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)